packages/sdk-gated/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@lens-protocol/sdk-gated",
+  "version": "1.2.0",
```

Comment:
what is the strategy for future releases? will sdk-gated be integrated into react-sdk and client-sdk and not available as a separate package anymore?

Reply:
that is still to be explored. 

Right now there is still a plan to somehow provide low-level gated SDK in case someone talks with the API directly (and at the moment 100% folks use API directly).

We need to see how the `@lens-protocol/client` gonna be adopted when released and make the proper decision. 

But no matter how awesome would be the new client sdk it would take some time to have good adoption across ecosystem.

TL;DR; we still want to release gated sdk as a separate beast.